ajax/form.php

```diff
@@ -30,14 +30,38 @@
  */
 
 include ('../../../inc/includes.php');
-Session::checkRight('entity', UPDATE);
```

Comment:
checkRight() outputs a HTML page on failure. This is not adapted with Ajax process.